docs/can-guides/topics/debugging.md

```diff
@@ -413,62 +418,62 @@ Clicking on a component in the tree will select it:
 
 <img src="../../docs/can-guides/images/devtools/panel-component-selected.png"
   class="bit-docs-screenshot"
-  alt="The CanJS Devtools Panel ViewModel Editor"
+  alt="The CanJS Devtools Panel Observable Properties Editor"
   width="600px"/>
 
 If a component is selected in the Elements Panel, it will automatically be selected in the CanJS Panel. If an element that is not a component is selected, its closest parent component will be selected automatically, if there is one.
 
-### ViewModel Editor
+### Observable Properties Editor
 
-As mentioned above, once a component is selected, its ViewModel will be displayed in the ViewModel Editor in the sidebar.
+As mentioned above, once a component is selected, its Observable Properties will be displayed in the Observable Properties Editor in the sidebar.
```

Comment:
Can you lowercase

> ...its observable properties will be...

This can remain uppercase since it is a proper noun:

> Observable Properties Editor